packages/onboarding-ui/src/forms/AdminInfoForm/AdminInfoForm.tsx

```diff
@@ -37,6 +41,7 @@ const AdminInfoForm = ({
   validateUsername,
   validateEmail,
   validatePassword,
+  keepPosted,
```

Comment:
I believe it's nice to be explicit on default values. It helps both TypeScript (not dealing with `undefined` type) and the component user (reading the default value as Storybook).

```suggestion
  keepPosted = false,
```

Reply:
Indeed